### PR TITLE
AFNetworking: Fix static analyzer issues.

### PR DESCRIPTION
**Pull request by Ner1Co**
For some reason, 4a49ba0 solved the static analyzer alone.
Should we add an assert anyhow?

### CODE REVIEW

**Comment by Ner1Co** [comment]
@dlahyani @StatusReport

File: AFNetworking/AFURLRequestSerialization.m [hunk 1]
```diff
@@ -655,10 +655,10 @@ - (void)appendHTTPBodyPart:(AFHTTPBodyPart *)bodyPart;
 #pragma mark -
 
 @interface AFStreamingMultipartFormData ()
-@property (readwrite, nonatomic, copy) NSMutableURLRequest *request;
-@property (readwrite, nonatomic, assign) NSStringEncoding stringEncoding;
-@property (readwrite, nonatomic, copy) NSString *boundary;
-@property (readwrite, nonatomic, strong) AFMultipartBodyStream *bodyStream;
+@property (readonly, nonatomic, strong) NSMutableURLRequest *request;
+@property (readonly, nonatomic, assign) NSStringEncoding stringEncoding;
+@property (readonly, nonatomic, strong) NSString *boundary;
```
Comment (StatusReport) [inline AFNetworking/AFURLRequestSerialization.m]: This must be a `copy` (make sure you understand why).

Comment (StatusReport) [reply AFNetworking/AFURLRequestSerialization.m]: However if it's `readonly` and you access by field it doesn't matter.

**Comment by dlahyani** [comment]
[Fix AFNetworking static analyzer warnings.](https://trello.com/c/hMc9liqy/9-fix-afnetworking-static-analyzer-warnings)

**Comment by dlahyani** [comment]
[Fix AFNetworking static analyzer warnings.](https://trello.com/c/hMc9liqy/9-fix-afnetworking-static-analyzer-warnings)

**Pull request merged**